### ISSUE TITLE
qtrigger doesn't start the cycle from the beginning

### ISSUE THREAD

**Issue by Zalastax** [tidalcycles/Tidal#1028]
Tidal version: 1.9.2

Code:
```Haskell
do
    let bd = "bd2!4"
    let hh = "hh!4"
    p "drums" $ qtrigger "drums" $ seqP [
        (0, 20, sound bd ),
        (20, 30, sound hh )]
```

Expectation from https://tidalcycles.org/docs/reference/composition/#seqp is that qtrigger can be used to locally reset time to 0.
When I try the code above, it does not seem to start from the beginning. I need to use `resetCycles` to start from cycle 0.

There's also https://tidalcycles.org/docs/patternlib/howtos/startpattern/ but those docs are not so clear.

**Comment by mindofmatthew** [comment]
I just tried to reproduce this on 1.9.4 and it seemed to work correctly. It generated 80 `bd2` events every quarter-cycle followed by 40 `hh` events. Could you try upgrading and seeing if that fixes the issue?

**Comment by Zalastax** [comment]
It is fixed in 1.9.4 and I can now use qtrigger without repeating the pattern name. Thank you very much!
